apps/meteor/client/components/RoomAutoComplete/Avatar.tsx

```diff
@@ -1,16 +1,17 @@
+import { RoomType } from '@rocket.chat/apps-engine/definition/rooms';
 import { Options } from '@rocket.chat/fuselage';
 import React, { FC } from 'react';
 
 import RoomAvatar from '../avatar/RoomAvatar';
 
 type AvatarProps = {
 	value: string;
-	type: string;
+	type: RoomType;
```

Comment:
Please, use `IRoom['type']` from `@rocket.chat/core-typings`. This type is from apps-engine.